web/sites/default/config/core.base_field_override.node.service_location.status.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 69db39b1-25ce-4da0-b0ed-6eed98a2c61d
+langcode: en
+status: true
+dependencies:
+  config:
+    - node.type.service_location
+id: node.service_location.status
```

Comment:
@jookyg, why is this field in here?

Reply:
@joshuami That's the Published checkbox, which I think we're disabling in custom code(?). I'm not sure why that might have been added in this case, but it exists on a few other types: Alert, Information, Notification, Page, and Top Search. I've compared the edit forms between those content types and the others, and can't see a difference in functionality. Do you want me to try removing it and see how it behaves?